nginx/assets/monitors/4xx.json

```diff
@@ -0,0 +1,32 @@
+{
+	"name": "[NGINX] 4xx Errors",
+	"type": "query alert",
+	"query": "avg(last_4h):anomalies(avg:NGINX.upstream.peers.responses.4xx{*}.as_count(), 'basic', 2, direction='above', alert_window='last_15m', interval=60, count_default_zero='true') >= 1",
```

Comment:
4 hours seems like a high time frame to alert on (for this and the remaining monitors) - would a shorter time window be more useful?

Reply:
how is that metric tagged? it's unlikely that we'll want to use the avg space aggregator. If it's a count metric would `sum` make more sense?

Reply:
how is that metric tagged? it's unlikely that we'll want to use the avg space aggregator. If it's a count metric would `sum` make more sense?

Reply:
@sanjuashok Shorter timeframe makes sense - updated to 1h

@remh These metrics are gauges (PR updated to reflect this) and should be tagged by `upsteam` name.
@hithwen, could you confirm the tagging scheme?

Reply:
Updated to include (and split by) `upstream` tag